docs/code_ref/noise.rst

```diff
@@ -0,0 +1,8 @@
+*****
+noise
+*****
+
+This package implements functions for noise level estimation.
+
+.. automodapi:: sunkit_image.noise
+   :no-inheritance-diagram:
```

Comment:
Why is this here?

Reply:
No idea I did not do anything to the noise module